neuromation/api/disks.py

```diff
@@ -0,0 +1,75 @@
+from dataclasses import dataclass
+from enum import Enum
+from typing import Any, AsyncIterator, Mapping
+
+from yarl import URL
+
+from .config import Config
+from .core import _Core
+from .utils import NoPublicConstructor
+
+
+@dataclass(frozen=True)
+class Disk:
+    id: str
+    storage: int  # In bytes
+    owner: str
+    status: "Disk.Status"
+    _cluster: str
```

Comment:
Why is it underscored?

Reply:
At first I thought that this is some kind of internal data (as it isn't returned from server), but now I see no reason to underscore this field.